### ISSUE TITLE
Is printing supported?

### ISSUE THREAD

**Issue by kaimast** [flathub/org.mozilla.Thunderbird#134]
I'm not sure if this is a limitation of flatpak. When I try to print an email Thunderbird will just crash. 

The crash happens before the printing dialog is shown, so I assume it fails to communicate with CUPS for some reason.

**Comment by Erick555** [comment]
Yes, it's supported and works just fine for me. What's your distro, desktop environment and flatpak version? I think flatpak 1.8.2 had some fix for cups which may be relevant.

**Comment by kaimast** [comment]
Actually, it might be a Wayland error? The only thing Thunderbird prints on the console before crashing is the following:

```Gdk-Message: 13:32:47.839: Error 71 (Protocol error) dispatching to Wayland display.```

**Comment by Erick555** [comment]
I didn't tried wayland so it's possible that it doesn't work there. Did you forced TB to run natively on wayland or does it run on xwayland?

**Comment by kaimast** [comment]
> I didn't tried wayland so it's possible that it doesn't work there. Did you forced TB to run natively on wayland or does it run on xwayland?

It's forced to run natively.

**Comment by Erick555** [comment]
Can you try on xwayland? Maybe it's some app issue.

**Comment by ovari** [comment]
Have you tried using [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal)?

In `Device`, is `All devices (e.g. webcam)` enabled?

Thank you

**Comment by kaimast** [comment]
It wasn't enabled on but enabling it did not seem to fix it.

**Comment by kaimast** [comment]
> Can you try on xwayland? Maybe it's some app issue.

Just ran without MOZ_ENABLE_WAYLAND set and it seems to work.

**Comment by Erick555** [comment]
It may be app issue with wayland then. I wonder if it's reproducible with non-flatpak version.

**Comment by Erick555** [comment]
@kaimast do you have idea if this issue still occurs with latest release?
